android/quest/src/main/java/org/smartregister/fhircore/quest/ui/report/measure/MeasureReportViewModel.kt

```diff
@@ -150,11 +153,11 @@ constructor(
 
     when (event) {
       is MeasureReportEvent.OnSelectMeasure -> {
-        measureReportConfig.value = event.measureReportConfig
+        measureReportConfig.value = event.measureReportConfig?.get(0)
```

Comment:
Any reasons why this is using a list? Can we extract the needed list item (the item at index 0)to it's own variable and reuse it?